static/index.js

```diff
@@ -0,0 +1,82 @@
+let reviews = [];
+
+let sum = 0;
+
+if(localStorage.getItem("reviews") !== null) reviews = JSON.parse(localStorage.getItem("reviews"));
```

Comment:
This is a thing of personal preference, but I am going to share mine:
I prefer always having curly braces to define the scope of the if statement block.
This way is more explicit and reduces the chance of errors.

Reply:
Thx for the advice.